app/authentication/user.py

```diff
@@ -8,3 +8,6 @@ def __init__(self, user_id, user_ik):
             self.user_ik = user_ik
         else:
             raise ValueError('No user_id or user_ik found in session')
+
+    def get_id(self):
+        return self.user_id
```

Comment:
I don't think we need this anymore but may be good to leave in for consistency across the user_id?

Reply:
Also technically no, but according to the spec Flask user classes are supposed to set `self.id` or implement `get_id`.